package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "eslint-config-supaapps",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
^^

Reply:
current github actions pipeline only publish version on tags, so let's revert this and do a separate version & tag on main later